src/VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -205,54 +205,5 @@
                 "PatternUrl": "/images/pattern-live.svg"
             }
         ]
-    },
-    "DefaultMainMenuState": {
```

Comment:
Should we remove these?

Reply:
It was in obsolete dev. There isn't now.